docs_src/microservices/application/V2Migration.md

```diff
@@ -0,0 +1,296 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
+    For the EdgeX 2.0 (Ireland) release there are many backward breaking changes. These changes require custom Application Services and custom profiles (app-service-configurable) to be migrated. This section outlines the necessary steps for this migration.
+
+## Custom Application Services
+
+### Configuration
+
+The migration of any Application Service's configuration starts with migrating configuration common to all EdgeX services. See the [V2 Migration of Common Configuration](../../configuration/V2MigrationCommonConfig) section for details. The remainder of this section focuses on configuration specific to Application Services.
```

Comment:
I get a 404 err for all of the links in this doc

Reply:
Yea, the links render differently in GitHub pages. I just verified it does redirect correctly.

FYI, we use a local `mkdocs-material`  tool to render the pages so we can verify these issue as we develop the pages. Checkout the README for edgex-docs here: https://github.com/edgexfoundry/edgex-docs/blob/master/README.md

If you want to validate the rendering (which I encourage) you can clone my branch for the PR and then run the `mkdocs-material` tool and view the rendered pages in your browser.